<p>Spring Controller Package Path Rule</p>
<h2>Noncompliant Code Example</h2>
<pre>
package xx.nocontroller.domain;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RestController;

@Controller
public class AController {
}

</pre>
<h2>Compliant Solution</h2>
<pre>
package xx.controller.domain;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RestController;

@Controller
public class AController {
}

</pre>
